docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,128 @@ platforms: ['android', 'ios', 'tvos']
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
-import { ConfigReactNative, ConfigPluginExample } from '~/components/plugins/ConfigSection';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { GithubIcon } from '@expo/styleguide-icons';
 
-The `expo-updates` library allows you to programmatically control and respond to new updates made available to your app.
+## Installation & Configuration
 
-## Installation
+`expo-updates` is a library that enables your app to manage remote updates to your application code. It communicates with the configured remote update service to get information about available updates.
+
+[EAS Update](/eas-update/introduction) is a hosted service that manages and serves updates, and includes tools to simplify installation and configuration of this library. To get started with EAS Update, follow the [Getting Started Guide](/eas-update/getting-started/).
```

Comment:
```suggestion
[EAS Update](/eas-update/introduction) is a hosted service that manages and serves updates, and includes tools to simplify installation and configuration of `expo-updates` library. Follow the instructions in the [EAS Update Get started](/eas-update/getting-started/) guide.
```

Reply:
I agree this is tough one. I am happy with the changes you've added.

Also, I think if we want to convey that EAS Update is optional, maybe we can rephrase this paragraph and the one following it to convey the same message here. It's often good to start with the most important information but in this case, after judging this, I think an important piece of info is that there are two ways to use this library. So may be it can be rephrased as:


```
There are two ways to configure `expo-updates` library: [EAS Update](/eas-update/introduction) and manual.

[EAS Update](/eas-update/introduction) is a hosted service that manages and serves updates, and includes tools to simplify installation and configuration of the `expo-updates` library. To get started with EAS Update, follow the instructions in the [Get started](/eas-update/getting-started/) guide.

Manual configuration is useful when the native code is also configured manually or with a different remote update service.
```

This makes the user aware that there are two different ways without the losing the context you've already added.

Reply:
Updated once more to simplify.